authnz/src/main/java/io/stargate/auth/TargetCell.java

```diff
@@ -0,0 +1,64 @@
+package io.stargate.auth;
+
+import io.stargate.db.datastore.query.Value;
+import io.stargate.db.datastore.query.WhereCondition;
+
+public class TargetCell {
```

Comment:
Finally realized I need something more generic than a list of Strings for handling the what's being filtered. Naming is not my strong suit so up for suggestions on this.

Reply:
maybe `KeyValue`? Not ideal, but, if I'm not mistaken, the term "cell" means a non-key values within a row in C*

Reply:
`TypedKeyValue`? :)

Reply:
sgtm :+1: